### PR TITLE
Graft of 'invert_serial_rx' onto next.

### PR DESCRIPTION
**Pull request by mikeller**
I wasn't sure if we'd want to leave

```uint8_t failsafeMode:7;
uint8_t invertedSerial:1;```

the way it is, or move `invertedSerial` into a better spot and do a conversion. Let me know.

### CODE REVIEW

File: radio/src/telemetry/frsky.cpp [hunk 1]
```diff
@@ -622,6 +631,15 @@ void telemetryInit(void)
   else {
     telemetryPortInit(FRSKY_SPORT_BAUDRATE);
   }
+
+#if defined(REVX)
```
Comment (bsongis) [inline radio/src/telemetry/frsky.cpp]: I remember it introduced a compilation error on VC++, it should be:
```
#if defined(REVX) && !defined(SIMU)
```

Comment (mikeller) [inline radio/src/telemetry/frsky.cpp]: Drats I can totally see why. 'learn how to build and use the simulator' is somewhere on my TODO list.

**Comment by bsongis** [comment]
Thanks!

**Comment by mikeller** [comment]
De rien.

**Pull request merged**